classpath/javax/net/ssl/X509KeyManager.java

```diff
@@ -1,21 +1,30 @@
 package javax.net.ssl;
 
+// import library for interface X509KeyManager
 import java.security.cert.X509Certificate;
 import java.security.Principal;
 import java.net.Socket;
 import java.security.PrivateKey;
+
```

Comment:
Unless there is special things that we do with our implementation, I personally don't think we need these comments. Users typically can lookup and/or already knows about these classes/interfaces if they are using it. Otherwise they can reference the java docs online.